app/components/CustomCard/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ *
+ * CustomCard
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import T from '@components/T';
+import { Card } from 'antd';
+import styled from 'styled-components';
+
+const StyledCard = styled(Card)`
+  && {
+    margin: 20px 0;
+    max-width: 900px;
```

Comment:
since this is a reusable component, let's get the maxWidth from props and have a default prop of 900